<?xml version="1.0"?>
<launch>

  <!-- lanedetector -->
  <node name="lanedetector" pkg="deepracer_av" type="lanedetector.py" respawn="true"/>

  <!-- lanekeeper -->
  <node name="lanekeeper" pkg="deepracer_av" type="lanekeeper.py" respawn="true"/>

  <!-- lane visualizer-->
  <node name="lane_visualizer" pkg="deepracer_av" type="lane_visualizer.py" respawn="true"/>

</launch>
